### ISSUE TITLE
Scanner does not correctly handle --rules and --group_rules flags

### ISSUE THREAD

**Issue by gbrindisi** [GoogleCloudPlatform/forseti-security#234]
(I'm not sure this is a bug, sorry if I'm wrong)

When I try to run the scanner with a sample rule I get:

```
$ forseti_scanner --rules ../forseti-evaluation/sample_rule.yaml
2017-04-19 13:40:50,783 __main__     INFO     main Initializing the rules engine:
Using rules: ../forseti-evaluation/sample_rule.yaml
Provide a rules file. Use "forseti_scanner --helpfull" for help.
```

Looks like the scanner is expecting both `--rules` and `--group_rules` flags, otherwise will exit.

I've found the affected code is in `scanner.py` ([here](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/scanner/scanner.py#L78)):

```
    if not FLAGS.rules or not FLAGS.group_rules:
        print 'Provide a rules file. Use "forseti_scanner --helpfull" for help.'
        sys.exit(1)
```

Testing with:

```
$ forseti_scanner --rules ../forseti-evaluation/sample_rule.yaml
```

At runtime the `not FLAGS.rules` is `False` while `not FLAGS.group_rules` is `True`, thus the whole condition resolve to `True`.

Is the `--group_rules` flag mandatory now? Otherwise maybe should be `and` instead of `or`?

**Comment by blueandgold** [comment]
Good catch Gianluca!  The group_rules should not be mandatory.  We might also have to think of a better way to configure which rules to run.  In the meantime, do you want to submit a PR to fix this, by reverting it back to its previous state?
```
    if not FLAGS.rules:
        print 'Provide a rules file. Use "forseti_scanner --helpful" for help.'
        sys.exit(1)
```
https://github.com/GoogleCloudPlatform/forseti-security/blob/5c1510b5e3cbcd56938c62d808a99eb380e5f68d/google/cloud/security/scanner/scanner.py

**Comment by carise** [comment]
I have a fix for this coming soon.

**Comment by carise** [comment]
Actually Gianluca, you can take the fix. (P.s. The help text should still say "helpfull", not "helpful")

**Comment by carise** [comment]
I've fixed this issue in #235 to prevent down time. Thanks!
